appveyor.yml

```diff
@@ -30,8 +30,6 @@ install:
   - 7z x clang.tar.bz2
   - 7z x clang.tar
   - set PATH=clang\bin;%PATH%
-  - llvm-config --version
```

Comment:
We don't need to download Clang anymore, right? You could remove the previous lines too if we don't.

Reply:
we need to have clang for the integration test

Reply:
Ah, right!